### PR TITLE
BlueMarbleGeographics.GlobalMapper version 24

### PR DESCRIPTION
**Pull request by TechnicaVivunt**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/85504)

### CODE REVIEW

**Review by Trenly — CHANGES_REQUESTED** [review]
I'm a bit confused by this package. It appears that this doesn't install Global Mapper itself, but rather this appears to be the Installer for the Installer for Global Mapper?

**Comment by TechnicaVivunt** [comment]
Yeah that's definitely odd; I will go ahead and switch this to .exe version of the installer instead since I know it's expected behavior.

File: manifests/b/BlueMarbleGeographics/GlobalMapper/24.00.0000/BlueMarbleGeographics.GlobalMapper.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: BlueMarbleGeographics.GlobalMapper
+PackageVersion: 24.00.0000
+PackageLocale: en-US
+Publisher: Blue Marble Geographics
+PackageName: Global Mapper
+License: Proprietary
+Copyright: 'Contact: geohelp@bluemarblegeo.com'
```
Comment (Trenly) [inline manifests/b/BlueMarbleGeographics/GlobalMapper/24.00.0000/BlueMarbleGeographics.GlobalMapper.locale.en-US.yaml:10]: This doesn't seem like a copyright

File: manifests/b/BlueMarbleGeographics/GlobalMapper/24.00.0000/BlueMarbleGeographics.GlobalMapper.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+PackageIdentifier: BlueMarbleGeographics.GlobalMapper
+PackageVersion: 24.00.0000
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://www.bluemarblegeo.com/downloads/global-mapper/global-mapper-24_0-x64.exe
+  InstallerSha256: AC367235DA4EE322554786300C9492D3ED125D3EB3B54458DE08C792F39380FB
+  InstallerSwitches:
+    Silent: /S
```
Comment (Trenly) [inline manifests/b/BlueMarbleGeographics/GlobalMapper/24.00.0000/BlueMarbleGeographics.GlobalMapper.installer.yaml:12]:
```suggestion
    Silent: /S
  ProductCode: '{277751E5-AA45-48BE-A78C-685205FB6196}'
  AppsAndFeaturesEntries:
    - DisplayName: Global Mapper 24.0 64-bit
      ProductCode: '{277751E5-AA45-48BE-A78C-685205FB6196}'
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by Trenly — APPROVED** [review]

**Review by Trenly — APPROVED** [review]

**Pull request merged**